streams/test-utils/src/main/java/org/apache/kafka/streams/test/ConsumerRecordFactory.java

```diff
@@ -246,7 +286,23 @@ public void advanceTimeMs(final long advanceMs) {
     public ConsumerRecord<byte[], byte[]> create(final String topicName,
```

Comment:
Nit introduced in this PR, but can you fix it: missing in JavaDoc comment: `Does not auto advance internally tracked time.`

Reply:
ack.